pingora/Cargo.toml

```diff
@@ -29,7 +29,7 @@ pingora-load-balancing = { version = "0.5.0", path = "../pingora-load-balancing"
 pingora-proxy = { version = "0.5.0", path = "../pingora-proxy", optional = true, default-features = false }
 pingora-cache = { version = "0.5.0", path = "../pingora-cache", optional = true, default-features = false }
 
-# Only used for documenting features, but doesn't work in any other dependency 
+# Only used for documenting features, but doesn't work in any other dependency
```

Comment:
I'd avoid making any changes that aren't strictly required when contributing changes upstream.